server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Mailbox.scala

```diff
@@ -67,29 +69,39 @@ case class DelegatedNamespace(user: Username) extends MailboxNamespace {
 }
 
 object SortOrder {
+  private val inboxOrder : UnsignedInt = 10L
+  private val archiveOrder : UnsignedInt = 20L
+  private val draftOrder : UnsignedInt = 30L
+  private val outboxOrder : UnsignedInt = 40L
+  private val sentOrder : UnsignedInt = 50L
+  private val trashOrder : UnsignedInt = 60L
+  private val spamOrder : UnsignedInt = 70L
+  private val templatesOrder : UnsignedInt = 80L
+  private val restoredMessageOrder : UnsignedInt = 90L
+  private val defaultOrder : UnsignedInt = 1000L
   private val defaultSortOrders = Map(
-      Role.INBOX -> SortOrder(10L),
-      Role.ARCHIVE -> SortOrder(20L),
-      Role.DRAFTS -> SortOrder(30L),
-      Role.OUTBOX -> SortOrder(40L),
-      Role.SENT -> SortOrder(50L),
-      Role.TRASH -> SortOrder(60L),
-      Role.SPAM -> SortOrder(70L),
-      Role.TEMPLATES -> SortOrder(80L),
-      Role.RESTORED_MESSAGES -> SortOrder(90L))
-    .withDefaultValue( SortOrder(1000L))
+      Role.INBOX -> SortOrder(inboxOrder),
```

Comment:
come on, we asked the opposite not so long ago

Reply:
What was the opposite ? i have not follow the discussion.
I extracted the creation of the UnsignedInt because it caused some issues during some macro expansion

Reply:
@mbaechler : can you explain again please? tried to do some archeology but there is so many comments I'm not sure to get which one you refer to